webapp/install_mattermost_webapp.sh

```diff
@@ -10,13 +10,12 @@ cd mattermost-webapp
 
 if [ ! -d .git ]; then
   git init
-  git remote add -f origin https://github.com/mattermost/mattermost.git
-  git config core.sparseCheckout true
-  echo "webapp/channels\nwebapp/platform/types\nwebapp/platform/client" >> .git/info/sparse-checkout
+  git config --local uploadpack.allowReachableSHA1InWant true
+  git remote add origin https://github.com/mattermost/mattermost.git
+  git fetch --depth=1 origin $COMMITHASH
+  git reset --hard FETCH_HEAD
 fi
 
-git fetch --depth=1 origin master
-git checkout -f $COMMITHASH
```

Comment:
I think we still need to cover this case. Keeping fetch + reset out of the if block should hopefully do it.

Reply:
Yah, you're right, missed that. Thanks.